frontend_v2/src/app/components/utility/selectphase/selectphase.component.ts

```diff
@@ -51,6 +51,16 @@ export class SelectphaseComponent implements OnInit, OnChanges {
    */
   selectedPhase: any = null;
 
+  /**
+   * Is phase selected
+   */
+  isPhaseSelected = false;
+
+  /**
+   * Is phase selected
+   */
+  isPhaseSplitSelected = false;
```

Comment:
Instead of declaring new variables for just to hide the label of dropdown, you can use the existing variable like `phaseName` and `selectedPhaseSplit` and check if they are not empty string. @Kajol-Kumari